### PR TITLE
sankey

### PR DESCRIPTION
**Pull request by jdamiba**

### CODE REVIEW

**Comment by jdamiba** [comment]
This PR replaces https://github.com/plotly/documentation/pull/1604

**Comment by nicolaskruchten** [comment]
@Mahdis-z can you get this PR updated for inclusion into this repo? basically without the `api_create` calls and following the same pattern as the other files :)

File: r/2017-05-19-sankey.Rmd [hunk 1]
```diff
@@ -91,8 +91,6 @@ p <- plot_ly(
     xaxis = list(showgrid = F, zeroline = F),
     yaxis = list(showgrid = F, zeroline = F)
   )
-
-p
```
Comment (nicolaskruchten) [inline r/2017-05-19-sankey.Rmd]: why was `p` removed here?

**Comment by nicolaskruchten** [comment]
ah, doh, I see that @jdamiba has updated the PR already for the new pattern! nice job Joe!

@Mahdis-z then please review this PR to make sure it matches your intention from the previous one it replaces :)

File: r/2017-05-19-sankey.Rmd (hunk 1 above)
Comment (nicolaskruchten) [reply r/2017-05-19-sankey.Rmd]: @jdamiba I think this quesion is for you

Comment (jdamiba) [reply r/2017-05-19-sankey.Rmd]: I removed the `p` because the `results = 'hide'` flag is present within this example code block.

Comment (nicolaskruchten) [reply r/2017-05-19-sankey.Rmd]: Oh I see. Weird, but ok.

**Comment by Mahdis-z** [comment]
> ah, doh, I see that @jdamiba has updated the PR already for the new pattern! nice job Joe!
> 
> @Mahdis-z then please review this PR to make sure it matches your intention from the previous one it replaces :)

good job @jdamiba thx, sounds good to me

**Pull request merged**